src/index.js

```diff
@@ -1,25 +1,25 @@
+import { getOptions, isUrlRequest } from 'loader-utils';
+import postcss from 'postcss';
+import postcssPkg from 'postcss/package.json';
 /*
   MIT License http://www.opensource.org/licenses/mit-license.php
   Author Tobias Koppers @sokra
 */
 import validateOptions from 'schema-utils';
-import postcss from 'postcss';
-import postcssPkg from 'postcss/package.json';
-
-import { getOptions, isUrlRequest } from 'loader-utils';
+import satisfies from 'semver/functions/satisfies';
```

Comment:
Let's use public API, no need hardcore path to function, it can be changed in theory

Reply:
this is public https://www.npmjs.com/package/semver The readme, points to this for smaller footprints